runway/hooks/staticsite/auth_at_edge/domain_updater.py

```diff
@@ -0,0 +1,99 @@
+"""User Pool Client Domain Updater."""
+import logging
+# pylint: disable=unused-import
```

Comment:
this should be in-line so it does not apply to the entire file, only a single line